components/org.wso2.micro.integrator.extensions/org.wso2.micro.integrator.probes/src/main/java/org/wso2/micro/integrator/probes/ReadinessResource.java

```diff
@@ -43,6 +46,7 @@
     private static final String HTTP_SC = "HTTP_SC";
     private static String CACHED_RESPONSE = "";
     private static int CACHED_RESPONSE_CODE;
+    private static Boolean HOT_DEPLOYMENT = null;
```

Comment:
This is not a final variable , it shouldn't be in caps lock ?

Reply:
Removed the variable since the logic is changed now.